classes/class-plugin.php

```diff
@@ -113,6 +113,9 @@ public function __construct() {
 
 		spl_autoload_register( array( $this, 'autoload' ) );
 
+		// Load Action Scheduler.
+		require_once $this->locations['dir'] . '/vendor/woocommerce/action-scheduler/action-scheduler.php';
```

Comment:
@tharsheblows There's a risk of conflict if this package was loaded by another plugin. Should this be preceded with `function_exists` call (or something similar) to only load this class if it has not been loaded by another plugin earlier?

Reply:
@bartoszgadomski Action Scheduler works when included multiple times! It's a clever library – it loads the most recent version requested so all of the functions expected are available. Eg latest version does it [like this ](https://github.com/woocommerce/action-scheduler/blob/trunk/action-scheduler.php)